Resources/views/PlatformUI/shell.html.twig

```diff
@@ -45,7 +45,15 @@
         <p>{{ 'javascript.disabled.required'|trans }}</p>
         <p><strong>{{ 'javascript.enable'|trans }}</strong></p>
     </noscript>
-    <script src="{{ asset( "bundles/ezplatformuiassets/vendors/yui3/build/yui/yui-min.js" ) }}"></script>
+   {% javascripts
+        'bundles/bazingajstranslation/js/translator.min.js'
+        'js/translations/config.js'
+    %}
+        {# We only load the translations for the eZPlatformUIBundle JS files #}
+        <script src="{{ url('bazinga_jstranslation_js', { 'domain': 'eZPlatformUIBundle_js' }) }}"></script>
+        <script src="{{ asset_url }}"></script>
+        <script src="{{ asset( "bundles/ezplatformuiassets/vendors/yui3/build/yui/yui-min.js" ) }}"></script>
```

Comment:
this tag and the one for `bazinga_jstranslation_js` should be outside of the `{% javascripts %}` block, otherwise, the page embeds those twice in dev mode.